<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/newPrefix/back2.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>321</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/newPrefix/C:/Users/MainUser/Desktop/tech_groupproject/school-management-system/sign/assets/back2.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="login_Button">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 186, 50);</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="signup_Button">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>230</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 186, 50);</string>
        </property>
        <property name="text">
         <string>SIGNUP</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/b.qrc"/>
 </resources>
 <connections/>
</ui>
